cabot/metricsapp/models/elastic.py

```diff
@@ -193,20 +193,34 @@ def _parse_series(self, series, series_name=None):
         :param series_name: the name for the series ('key1.key2. ... .metric')
         :return: (series, (timestamp, datapoint)) pairs
         """
+
+        this_series_name = series_name
```

Comment:
supernit: can we remove `this_series_name` and change `series_name` in line 205 to `subseries_name`? I don't really like this_series_name, it's not very specific